app/src/main/java/com/sygic/travel/sdkdemo/detail/PlaceDetailActivity.kt

```diff
@@ -180,35 +214,20 @@ class PlaceDetailActivity : AppCompatActivity() {
         }
 
     private inner class Views internal constructor() {
-        internal var ivPhoto: ImageView
-        internal var tvName: TextView
-        internal var tvNameSuffix: TextView
-        internal var tvPerex: TextView
-        internal var tvDescription: TextView
-        internal var tvRating: TextView
-        internal var tvAddress: TextView
-        internal var tvPhone: TextView
-        internal var tvEmail: TextView
-        internal var tvAdmission: TextView
-        internal var tvOpeningHours: TextView
-        internal var fblTags: FlexboxLayout
-        internal var llReferencesList: LinearLayout
-
-        init {
-            ivPhoto = findViewById(R.id.iv_detail_photo) as ImageView
-            tvName = findViewById(R.id.tv_name) as TextView
-            tvNameSuffix = findViewById(R.id.tv_name_suffix) as TextView
-            tvPerex = findViewById(R.id.tv_perex) as TextView
-            tvDescription = findViewById(R.id.tv_description) as TextView
-            tvRating = findViewById(R.id.tv_rating) as TextView
-            tvAddress = findViewById(R.id.tv_address) as TextView
-            tvPhone = findViewById(R.id.tv_phone) as TextView
-            tvEmail = findViewById(R.id.tv_email) as TextView
-            tvAdmission = findViewById(R.id.tv_admission) as TextView
-            tvOpeningHours = findViewById(R.id.tv_opening_hours) as TextView
-            fblTags = findViewById(R.id.fbl_tags) as FlexboxLayout
-            llReferencesList = findViewById(R.id.ll_references_list) as LinearLayout
-        }
+        internal var ivPhoto: ImageView = findViewById(R.id.iv_detail_photo) as ImageView
+        internal var tvName: TextView = findViewById(R.id.tv_name) as TextView
+        internal var tvNameSuffix: TextView = findViewById(R.id.tv_name_suffix) as TextView
+        internal var tvPerex: TextView = findViewById(R.id.tv_perex) as TextView
+        internal var tvDescription: TextView = findViewById(R.id.tv_description) as TextView
+        internal var tvRating: TextView = findViewById(R.id.tv_rating) as TextView
+        internal var tvAddress: TextView = findViewById(R.id.tv_address) as TextView
+        internal var tvPhone: TextView = findViewById(R.id.tv_phone) as TextView
+        internal var tvEmail: TextView = findViewById(R.id.tv_email) as TextView
+        internal var tvAdmission: TextView = findViewById(R.id.tv_admission) as TextView
+        internal var tvOpeningHours: TextView = findViewById(R.id.tv_opening_hours) as TextView
+        internal var fblTags: FlexboxLayout = findViewById(R.id.fbl_tags) as FlexboxLayout
+        internal var llReferencesList: LinearLayout = findViewById(R.id.ll_references_list) as LinearLayout
+        internal var cbFavorite: CheckBox = findViewById(R.id.cb_favorite) as CheckBox
```

Comment:
Oh, that's Is, not Ls. Well, getter for a boolean property is prefixed with "is", so the property itself is not.